### PR TITLE
Add embroider app

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Doesn't exactly supersede: https://github.com/bgantzler/ember-mirage/pull/13
but gets us embroider coverage.

### CODE REVIEW

File: packages/ember-mirage/package.json [hunk 1]
```diff
@@ -28,8 +28,9 @@
     "prepack": "rollup --config"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.0.0",
-    "@embroider/macros": "^1.12.0",
+    "@embroider/addon-shim": "^1.8.6",
+    "@embroider/macros": "^1.13.1",
+    "ember-inflector": "^4.0.2",
```
Comment (NullVoxPopuli) [inline packages/ember-mirage/package.json:33]: @cah-brian-gantzler this addresses the error you saw

Comment (cah-brian-gantzler) [reply packages/ember-mirage/package.json:33]: thanks, was just looking into creating a PR for that.

File: test-apps/01-sample-app/app/app.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import Application from '@ember/application';
 
 import loadInitializers from 'ember-load-initializers';
-import config from 'ember-mirage-sample/config/environment';
+import config from 'test-app/config/environment';
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply test-apps/01-sample-app/app/app.js]: it's a config you'd have in the vite config -- an alias

[the remark this answers is not in the document]
Comment (cah-brian-gantzler) [reply test-apps/01-sample-app/app/app.js]: Ahh. We dont have a vite config yet do we? Or if you create one it works now? Any documentation I can read?

**Pull request merged**